setuptools/command/build_py.py

```diff
@@ -48,21 +49,50 @@ def finalize_options(self):
         if 'data_files' in self.__dict__:
             del self.__dict__['data_files']
 
-    def copy_file(  # type: ignore[override] # No overload, no bytes support
+    @overload  # type: ignore[override] # Truthy link with bytes is not supported, unlike supertype
+    def copy_file(
         self,
         infile: StrPath,
         outfile: StrPathT,
         preserve_mode: bool = True,
         preserve_times: bool = True,
         link: str | None = None,
-        level: object = 1,
-    ) -> tuple[StrPathT | str, bool]:
+        level: int = 1,
```

Comment:
This was typed as `object` because it's unused. However, distutils types it as `int`.

So either distutils should change it to `object` (or `Any`) to indicate that anything can be passed to `level` and that subclasses are *not* intended to implement it.
Or, if some subclasses are meant to implement it, then this should indeed be `level: int` even if unused.